etc/docker/dev/docker-compose-storage-externalmetadata.yml

```diff
@@ -1,4 +1,5 @@
-version: "3.5"
+version: '3.4'
```

Comment:
going down a version number?

Reply:
Fixed. I renamed the final commit (which concerns the `docker-compose`s) because it no longer fixes issue #5983, it only adds the `web1` container.